From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Fri, 26 Jul 2024 11:49:17 +0200
Subject: u-libc-hurd-amd64

https://github.com/rust-lang/libc/pull/3786

commit 3ff489613e871677693b39c93ddc208c2ec584fc

    hurd: Add missing struct __timeval for 64bit support
    
    This is the same as linux_like/linux/gnu's __timeval for ut_tv for 64bit
    architectures.
---
 vendor/libc-0.2.155/src/unix/hurd/mod.rs | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/vendor/libc-0.2.155/src/unix/hurd/mod.rs b/vendor/libc-0.2.155/src/unix/hurd/mod.rs
index 2701649..42493b8 100644
--- a/vendor/libc-0.2.155/src/unix/hurd/mod.rs
+++ b/vendor/libc-0.2.155/src/unix/hurd/mod.rs
@@ -450,6 +450,11 @@ s! {
         pub tv_nsec: __syscall_slong_t,
     }
 
+    pub struct __timeval {
+        pub tv_sec: i32,
+        pub tv_usec: i32,
+    }
+
     pub struct __locale_data {
         pub _address: u8,
     }
